tools/release-plugin.sh

```diff
@@ -42,6 +42,13 @@ if ! version_compare "$GH_VER" "2.21.2"; then
 	die "Your version of the GH CLI is out of date. Please upgrade your version$WITH and start again"
 fi
 
+# Make sure we're signed into the GitHub CLI.
+if ! gh auth status &> /dev/null; then
```

Comment:
I wonder if we should do like this
```suggestion
if ! gh auth status --hostname github.com &> /dev/null; then
```
since we only care about GH itself here and not any enterprise things.